hack/generate-in-docker.sh

```diff
@@ -17,8 +17,9 @@ if hash getenforce 2> /dev/null && getenforce | grep 'Enforcing' > /dev/null; th
   VOLUME_OPTIONS=":Z"
 fi
 
+# shellcheck disable=SC2068
 docker run \
     --rm \
     -u="$(id -u "$USER")":"$(id -g "$USER")" \
     -v "${SCRIPTDIR}/..:/go/src/github.com/coreos/prometheus-operator${VOLUME_OPTIONS}" \
-    po-jsonnet make "${MFLAGS[@]}" generate
+    po-jsonnet make ${MFLAGS[@]} generate
```

Comment:
What was the issue here?

Reply:
in this case we actually want the flags to expand and if there is nothing, with quotes the command will be `make "" generate`, and `""` is not a valid make target.